src/scripts/browserDeviceProfile.js

```diff
@@ -924,7 +924,7 @@ export default function (options) {
 
     profile.ContainerProfiles = [];
 
-    if (browser.tizen) {
+    if (browser.tizen && browser.tizenVersion < 6) {
```

Comment:
According to https://github.com/jellyfin/jellyfin-web/issues/7231#issuecomment-3474161085, QN90B is `6.5`.

You can omit `browser.tizen &&`, because `browser.tizenVersion` is only defined when `browser.tizen` is `true`.